.github/workflows/build_packages.yaml

```diff
@@ -35,10 +36,14 @@ jobs:
         shell: bash
         working-directory: source
         run: |
-          vsn="$(./pkg-vsn.sh)"
-          pre_vsn="$(echo $vsn | grep -oE '^[0-9]+.[0-9]')"
-          old_vsns="$(git tag -l "v$pre_vsn.[0-9]" | xargs echo -n | sed "s/v$vsn//")"
-          echo "::set-output name=old_vsns::$old_vsns"
+          ce_vsn="$(./pkg-vsn.sh community)"
+          ee_vsn="$(./pkg-vsn.sh enterprise)"
+          ce_pre_vsn="$(echo $ce_vsn | grep -oE '^[0-9]+.[0-9]')"
+          ee_pre_vsn="$(echo $ee_vsn | grep -oE '^[0-9]+.[0-9]')"
```

Comment:
Q: couldn't the minor version number be more than one digit eventually (like in `5.10.0`)?

Reply:
thanks. fixed.
and you just helped to discovered a bug in 4.3
https://github.com/emqx/emqx/pull/6857